File: Leetcode/python/kids-with-the-greatest-number-of-candies.py
# https://leetcode.com/problems/kids-with-the-greatest-number-of-candies/

def kidsWithCandies(candies, extraCandies):
    maxCandies = max(candies)
    candyList = []
    for candy in candies:
        if candy + extraCandies >= maxCandies:
            candyList.append(True)
        else:
            candyList.append(False)
            
    return candyList

File: Leetcode/javascript/kids-with-the-greatest-number-of-candies.js
// https://leetcode.com/problems/kids-with-the-greatest-number-of-candies/

var kidsWithCandies = function(candies, extraCandies) {
    var maxCandies = Math.max.apply(null, candies)
    var candyList = []
    for (i=0; i < candies.length; i++) {
        if (candies[i] + extraCandies >= maxCandies){
            candyList.push(true)
        }
        else {
            candyList.push(false)
        }
    }    
    return candyList
    
};